__tests__/app.test.js

```diff
@@ -318,5 +318,84 @@ describe('app.js test suite', () => {
                 })
             })
         });
+
+        test('responds with articles sorted by specified column', () => {
+            return request(app)
+            .get('/api/articles?sort_by=author')
+            .expect(200)
+            .then(({body}) => {
+                
+                expect(body.articles).toHaveLength(12)
+                expect(body.articles).toBeSortedBy('author', {
+                    descending: true
+                })
+            })
+        });
+
+        test('responds with articles sorted by ascending when specified', () => {
+            return request(app)
+            .get('/api/articles?order=asc')
+            .expect(200)
+            .then(({body}) => {
+                
+                expect(body.articles).toHaveLength(12)
+                expect(body.articles).toBeSortedBy('created_at')
+            })
+        });
+
+        test('responds with articles filtered by topic, sorted by column_name, and in correct order when all are specified ', () => {
+            return request(app)
+            .get('/api/articles?topic=mitch&sort_by=author&order=asc')
+            .expect(200)
+            .then(({body}) => {
+                
+                expect(body.articles).toHaveLength(11)
+                expect(body.articles).toBeSortedBy('author')
+                body.articles.forEach((article) =>{
+
+                    expect(article.topic).toBe('mitch');
+                })
+            })
+        });
+
```

Comment:
Another status 200 might be good for a topic that does exist but does not have any reviews yet! Have a look in your test data file to see what topic you could use!